src/components/SlideUp/SlideUp.test.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {fireEvent, render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import SlideUp from './index';
+
+const slideUpRoot = document.createElement('div');
+slideUpRoot.setAttribute('id', 'slide-up-root');
+document.body.appendChild(slideUpRoot);
+
+describe('SlideUp component', () => {
+  test('renders with text passed in', () => {
+    render(<SlideUp close={() => {}}>Hello World</SlideUp>);
+    expect(screen.getByText('Hello World')).toBeTruthy();
+  });
+
+  test('calls close prop when component is clicked', () => {
+    const close = jest.fn();
+    const {getByRole} = render(<SlideUp close={close}>Hello World</SlideUp>);
```

Comment:
lets not do this destructuring from render, and grabbing by the Role. You can pass an `data-testid` if you want to grab the button.

Reply:
Thank you for the suggestion. I'm changing it to that in the next git push arriving shortly.